src/cookbooks/operational_monitoring.md

```diff
@@ -0,0 +1,120 @@
+# Introduction to Operational Monitoring
+
+Operational monitoring is a general term that refers to monitoring the health of software.
+
+However, Mozilla's internal Operational Monitoring tooling supports a couple of specific use cases:
+
+1. Monitoring build over build. This is typically for Nightly where one build may contain changes that a previous build doesn't and we want to see if those changes affected certain metrics.
+2. Monitoring by submission date over time. This is helpful for a rollout in Release for example, where we want to make sure there are no performance or stability regressions over time as a new build rolls out.
+
+The monitoring dashboards produced for these use cases are available in [Looker](https://looker.com/).
```

Comment:
We can link directly to the operational monitoring folder in Looker:
```suggestion
The monitoring dashboards produced for these use cases are available in [Looker](https://mozilla.cloud.looker.com/folders/494).
```

Reply:
Sidenote: I seem to not have access to data in the Fission dashboard. Are those intentionally locked down?

Reply:
Yes, I don't have access either. I think you placed that dashboard in the folder, right @scholtzan? Here's what I see:
<img width="1318" alt="Screen Shot 2022-01-12 at 12 17 19 PM" src="https://user-images.githubusercontent.com/784781/149189499-a621cecf-5345-4463-a50c-afef4287eefc.png">

If I go to Folders > LookML Dashboards I see them at the bottom of the list.

Reply:
Thanks!